### PR TITLE
Update Regex for Issue #28

### LINKED ISSUE #28: Update regex to ignore additional white space before ++
@earl++
@earl ++
these both work but
@earl  ++ and more do not, we should fix it.

### PR DESCRIPTION
**Pull request by ashaw7697**
Closes #28

### CODE REVIEW

File: 9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
         <role_conditions/>
         <script><![CDATA[(function executeRule(current, previous /*null when async*/) {
 
-	var matches = current.text.match(/(?:<@)(.*?)(?=>\s?\+\+)/g);
+	var matches = current.text.match(/(?:<@)(.*?)(?=>\s*\+\+)/g);
```
Comment (chelming) [inline 9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml:32]: this doesn't take any spaces into account between the two plusses

Comment (chelming) [reply 9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml:32]: changing the regex to `/(?:<@)(.*?)(?=>\s*\+){2}/` should be pretty inclusive.

e.g., https://regex101.com/r/O3XsrE/2

Comment (chelming) [reply 9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml:32]: regex in this commit will fail on the top two use cases, as seen here https://regex101.com/r/nLj5fl/1

File: 9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
         <role_conditions/>
         <script><![CDATA[(function executeRule(current, previous /*null when async*/) {
 
-	var matches = current.text.match(/(?:<@)(.*?)(?=>\s*\+\+)/g);
+	var matches = current.text.match(/(?:<@)(.*?)(?=>\s*\+){2}/gm);
```
Comment (chelming) [inline 9c7718cddb229150791d8f8d13961979/update/sys_script_b91e6805dba29150791d8f8d13961933.xml:32]: ok, so, uh, my bad... this regex doesn't work with emca. It needs to be
`/(?:<@)(.*?)(?=>(?:\s*\+){2})/gm`

**Comment by chelming** [comment]
lgtm 😀 

@earlduque you wanna do the honors?

**Comment by earlduque** [comment]
Will pull into production later today

**Pull request merged**